### ISSUE TITLE
ZeroNet is blocked in China Now

### ISSUE THREAD

**Issue by kehao95** [HelloZeroNet/ZeroNet#1401]
I can't reach any peer on ZeroHello.

**Comment by funny110** [comment]
Just block all the trackers of the newest ZeroNet. use VPN help you get some peers, than the zite  will works well for you.

**Comment by HelloZeroNet** [comment]
Yes, they blocked the trackers that requires to collect new ips. Currently working on multiple solutions, one of them will be: https://github.com/HelloZeroNet/ZeroNet/issues/1396
